Formula/aws-okta.rb

```diff
@@ -0,0 +1,21 @@
+class AwsOkta < Formula
+  desc "Aws-vault like tool for Okta authentication"
```

Comment:
I think `authenticate with AWS using your Okta credentials` is a better description

Reply:
ok, will change